app/templates/projectModel.html

```diff
@@ -20,8 +20,12 @@
                 {{ form.name }}
                 <div class="inline"> {{ form.startDate }} - {{ form.endDate }} </div>
 
-                <div class="flex justify-end">
+                <div class="py-8">
+                    Contributors
+                    <div class="pt-1">{{ form.contributors }}</div>
+                </div>
 
+                <div class="flex justify-end ">
```

Comment:
I wonder if it would be better to create a separate button for adding contributors that has an image person on it with a plus symbol so it's more intuitive that this functionally is there

Reply:
Also it would be nice to populate the text area with existing contributors

Reply:
Existing contributors are automatically populated (minus the current user). I do agree, though that there is a more intuitive way to add contributors but Erik seemed to want the sharing functionality fairly quickly so I was going to update that in a future PR.